src/app/models/test-type.model.ts

```diff
@@ -15,4 +15,8 @@ export interface TestType {
   prohibitionIssued: boolean;
 
   testResult: string;
+
+  seatbeltInstallationCheckDate: boolean;
+  numberOfSeatbeltsFitted: number;
+  lastSeatbeltInstallationCheckDate: string | Date;
```

Comment:
Is this the correct type? Feels like it should be a nullable Date

Reply:
I went off of what the other fields were using, I don't think we've had something that uses a nullable date yet so not entirely sure how I would change it to match that?

Reply:
Looking at the data, the field appears to be "Were the seatbelts checked on this date" and if so it makes the other fields mandatory. I think lastSeatbeltInstallationCheckDate is a literal "The last time we checked the seatbelts".

I've gone on the assumption by the way these need to match the structure of how they are in the DB? If not then i'd happily rename it to something more useful but not sure how i'd point it to the right field